spec/requests/api/v1/api_provides_client_collection_of_products_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe "GET/api/v1/products" do
+    let!(:products){3.times{create(:product)}}
+
+    describe "successfully get list of products" do
+        before do
+            get "/api/v1/products"
+        end
+
+        it "expected to return a 200 response" do
+            expect(response).to have_http_status 200
+        end
+
+        it "expected to return all products" do
+            expect(response_json["products"].count).to eq 3
+        end
```

Comment:
Please keep the indentation standardized. 
You are using two spaces, but now tabs it seems. 
Make sure everyone in the group has the same setting in the code editor. Please ask your coach how to set this if you don't know. Otherwise a quick google search will probably solve this.